examples/k8sdeploy.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: heartbeat-proxy
+  name: heartbeat-proxy
+  namespace: monitoring
+spec:
+  ports:
+    - name: http
+      port: 80
+      protocol: TCP
+      targetPort: 8080
+  selector:
+    app: heartbeat-proxy
+  type: "ClusterIP"
+#  type: "LoadBalancer"
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: heartbeat-proxy
+  name: heartbeat-proxy
+  namespace: monitoring
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: heartbeat-proxy
+    spec:
+      containers:
+        - name: heartbeat-proxy
+          image: registry.gitlab.dm-drogeriemarkt.com/docker/opsgenie-heartbeat-proxy
```

Comment:
Could you please exchange the `image` for the autobuild one from https://hub.docker.com/r/traumfewo/opsgenie-heartbeat-proxy/